packages/ndla-ui/src/ResourceBox/ResourceBox.tsx

```diff
@@ -0,0 +1,194 @@
+/**
+ * Copyright (c) 2022-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+import React from 'react';
+import { breakpoints, fonts, mq, colors } from '@ndla/core';
+import BEMHelper from 'react-bem-helper';
+import { useTranslation } from 'react-i18next';
+import { Launch } from '@ndla/icons/common';
+import { LicenseByline } from '@ndla/licenses';
+import styled from '@emotion/styled';
+import Image from '../Image';
+
+const classes = new BEMHelper({
+  name: 'figure',
+  prefix: 'c-',
+});
```

Comment:
Denne brukes kun for å sette className på div og span i licensebyline. Du kan fint fjerne denne og erstatte med rein setting av className, siden du også bruker styled. Sjekk neste kommentar.